content/en/blog/_posts/2023-12-05-image-filesystem.md

```diff
@@ -0,0 +1,171 @@
+---
+layout: blog
+title: 'Image Filesystem: Configuring Kubernetes to store containers on a separate filesystem'
+date: 2023-12-05
+slug: kubernetes-separate-imagefilesystem
+---
+
+**Authors:** Kevin Hannon (Red Hat)
+
+A common issue in running/operating Kubernetes clusters is running out of disk space.
+When the node is provisioned, you should aim to have a good amount of storage space for the container runtime.  
+The container runtime usually writes to the `/var` partition.
+CRI-O, by default, writes its containers and images to `/var/lib/containers`, while Containerd writes its containers and images to `/var/lib/containerd`.
```

Comment:
```suggestion
CRI-O, by default, writes its containers and images to `/var/lib/containers`, while containerd writes its containers and images to `/var/lib/containerd`.
```

Reply:
I think it should be capitalized because Containerd is a project name.

Reply:
I don't know english rules anymore apparently.  Not sure which is correct..